src/upload-release-asset.js

```diff
@@ -26,6 +27,7 @@ async function run() {
       url: uploadUrl,
       headers,
       name: assetName,
+      label: assetLabel,
```

Comment:
Will action work fine if you won't set `asset_label`?

Reply:
yes. It may be good to add a github action to test this against the real github API, to avoid this kind of uncertainty.